lib/pages/chat_details/chat_details_edit.dart

```diff
@@ -404,12 +411,15 @@ class ChatDetailsEditController extends State<ChatDetailsEdit>
           );
         }
       },
-      (success) {
+      (success) async {
         if (success is UpdateAvatarGroupChatSuccess) {
           Logs().d(
             'ChatDetailsEdit::_handleUpdateGroupInfoOnEvents() - UpdateAvatarGroupChatSuccess',
           );
           _handleSaveIconOnUpdateSuccess();
+          await Future.delayed(_delayedUpdateAvatarDuration).then((_) {
```

Comment:
why is the delay needed?

Reply:
I don't know exactly avatar update successfully in `room`. So I wait a little time for it.

<img width="300" alt="Screenshot 2024-02-01 at 08 45 15" src="https://github.com/linagora/twake-on-matrix/assets/99852347/cd9c1203-0a52-4888-8883-20c010527268">